cardboard/templates/tools.html

```diff
@@ -140,25 +151,15 @@ <h1>Tools and References</h1>
             </div>
         </div>
 
-        <div class="card" >
-            <div class="card-body">
-                <div class="card-title"><h2>Useful Links</h2></div>
-                <div class="card-text">
-                    <ul>
-                        <li><a href='https://discord.gg/WRywM5Cjh7'>Discord invite link</a></li>
-                        <li><a href='https://docs.google.com/drawings/d/1-XH2WiauqBgDQrak0LURyWhSIKM2yGZjhpVKj6gX8Dc/edit?usp=sharing'>Cardinality connections web</a></li>
-                    </ul>
-                </div>
-            </div>
-        </div>
-
         <div class="card" >
             <div class="card-body">
                 <div class="card-title"><h2>MITMH Specific</h2></div>
                 <div class="card-text">
                     <ul>
+                        <li><a href="http://puzzles.mit.edu/tools.html">MITMH Puzzles Tools Page</a></li>
                         <li><a href='https://thetech.com/issues/140/43/pdf'>MIT Newspaper</a></li>
                         <li><a href='https://docs.google.com/spreadsheets/d/1vxO5hI2Uqr8rB5QOA3u0AOQYj6CSBeCyPVAylaZZvbg/edit?usp=sharing'>Useful Links</a></li>
+                        <li><a href='https://docs.google.com/drawings/d/1n0DcJpA4dCd63dUnqG2TFmd006FtBJEMrVVfDUPiVt4/edit'>Cardinality connections web</a></li>

```

Comment:
did you mean to add this?

Reply:
I was actually sorta torn whether I should include it, what do yall think?

Reply:
If we're expecting non-Cardinality people to use Cardboard, seems kinda weird that they'd be able to see the link at all. 

We could add it to a future Cardinality team website?

Reply:
I think it's okay to leave out since we have other ways of distributing it (Discord or email) and I feel like people are going to look for it there anyway. IMO we should feel comfortable putting stuff on here if we think it's useful for us even if it's not useful for other people, though.